### ISSUE TITLE
Map with overlaid pie charts demo, pie chart sizing

### ISSUE THREAD

**Issue by dhart31** [highcharts/highcharts#19702]
#### Expected behaviour
Modifying the plot options "minSize" and "maxSize" should change the range of possible pie chart sizes in the "map with overlaid pie charts" demo.

#### Actual behaviour
Pie chart sizes are unaffected by minSize/maxSize parameter values when they are not set within onPoint

#### Live demo with steps to reproduce
Moving the options into onPoint seems to fix the issue, as shown here: https://jsfiddle.net/6f3qdLwc/
However, its not clear to me why these parameters cannot be specified as generic options for the series outside of onPoint, since they are the same value for all points in the series. Is this intentional, or a bug?

#### Product version
Highmaps v11.1.0

**Comment by karolkolodziej** [comment]
Hi @dhart31! 
Thank you for reporting the issue! 

IMO this isn't correct because other options are merged and used correctly.

_Internal note:_
It looks like the problem starts when the `SeriesOnPointComposition` tries to calculate the `seriesTranslate`.
Consequently, the `getPxExtremes` has no access to correct series options and returns incorrect values. 
https://github.com/highcharts/highcharts/blob/3c9dc1ef922e2752670d5fbe6bd9bb706bce2c3c/ts/Series/Bubble/BubbleSeries.ts#L769-L794

**Comment by hubertkozik** [comment]
Yes, it seems to me, that these properties should work also on series object (and `plotOptions` too).

Also, I think this ticket should include improving API docs: https://api.highcharts.com/highmaps/series.pie.onPoint and https://api.highcharts.com/highmaps/series.pie (missing `minSize` and `maxSize`).

_Internal note:_
`SeriesOnPointComposition` is using `mapbubble` series methods, that's why `minSize` and `maxSize` are used in `pie` series.
